ui/theme/theme.go

```diff
@@ -55,19 +57,27 @@ func NewMyTheme(config *backend.ThemeConfig, themeFileDir string) *MyTheme {
 	m := &MyTheme{config: config, themeFileDir: themeFileDir}
 	m.defaultThemeFile, _ = DecodeThemeFile(bytes.NewReader(res.ResDefaultToml.StaticContent))
 	m.createThemeIcons()
+	if err := m.createThemeResources(); err != nil {
+		log.Printf("failed to create theme resources: %s", err.Error())
+	}
 	return m
 }
 
 func (m *MyTheme) Color(name fyne.ThemeColorName, _ fyne.ThemeVariant) color.Color {
 	// load theme file if necessary
 	if m.loadedThemeFile == nil || m.config.ThemeFile != m.loadedThemeFilename {
+		if m.config.ThemeFile == "" {
+			m.config.ThemeFile = "default.toml"
+		}
+
 		t, err := ReadThemeFile(path.Join(m.themeFileDir, m.config.ThemeFile))
```

Comment:
As stated above, the previous code was trying to read the themes directory as a theme file, since ThemeFile was an empty string.